metricbeat/docs/modules/http.asciidoc

```diff
@@ -5,8 +5,8 @@ This file is generated! See scripts/docs_collector.py
 [[metricbeat-module-http]]
 == http Module
```

Comment:
"http Module" looks weird in the docs due to capitalization. I think there was a way to define the title in `fields.yml`?